src/services.js

```diff
@@ -42,3 +43,11 @@ export async function queuePluginInstall(plugin, token, priority = 10) {
     data: { plugin, activate: true, queue: true, priority },
   }).catch((error) => 'failed');
 }
+
+export async function createProduct(data) {
```

Comment:
Can we add some error logic as well? Let's say if for some reason the call fails to add a new page, do we redirect them to `post-new.php?post_type=gift_card`?